ale/drivers/base.py

```diff
@@ -195,13 +264,51 @@ def detector_center(self):
     def spacecraft_name(self):
         return self._label['MISSION_NAME']
 
+    @property
+    def starting_detector_line(self):
+        return 1
+
+    @property
+    def starting_detector_sample(self):
+        return 1
+
+    @property
+    def detector_sample_summing(self):
+        return 1
+
+    @property
+    def detector_line_summing(self):
+        return self._label.get('SAMPLING_FACTOR', 1)
+
+    @property
+    def filter_number(self):
+        return self._label.get('FILTER_NUMBER', 0)
+
+
+class Spice():
+    @property
+    def metakernel(self):
+        pass
+
+    @property
+    def odtx(self):
+        return spice.gdpool('INS{}_OD_T_X'.format(self.ikid),0, 10)
+
+    @property
+    def odty(self):
+        return spice.gdpool('INS{}_OD_T_Y'.format(self.ikid), 0, 10)
+
+    @property
+    def odtk(self):
+        return spice.gdpool('INS{}_OD_K'.format(self.ikid),0, 3)
+
     @property
     def ikid(self):
         return spice.bods2c(self.instrument_id)
 
     @property
     def fikid(self):
-        fn = self._label.get('FILTER_NUMBER', 0)
+        fn = self.filter_number
```

Comment:
I was never enamored with this. Felt hacky, so please alter the whole fikid if that makes sense?